src/transactions/test/PathPaymentStrictSendTests.cpp

```diff
@@ -509,67 +509,107 @@ TEST_CASE("pathpayment strict send", "[tx][pathpayment]")
         });
     }
 
-    SECTION("send issuer missing")
+    SECTION("issuer missing")
     {
         auto market = TestMarket{*app};
         auto source = root.create("source", minBalance1);
         auto destination = root.create("destination", minBalance1);
         source.changeTrust(idr, 20);
         destination.changeTrust(usd, 20);
         gateway.pay(source, idr, 10);
-        for_versions_from(12, *app, [&] {
-            gateway.merge(root);
-            REQUIRE_THROWS_AS(source.pathPaymentStrictSend(destination, idr, 10,
-                                                           usd, 10, {}, &idr),
-                              ex_PATH_PAYMENT_STRICT_SEND_NO_ISSUER);
-            // clang-format off
-            market.requireBalances(
-                {{source, {{xlm, minBalance1 - 2 * txfee}, {idr, 10}, {usd, 0}}},
-                 {destination, {{xlm, minBalance1 - txfee}, {idr, 0}, {usd, 0}}}});
-            // clang-format on
-        });
-    }
 
-    SECTION("middle issuer missing")
-    {
-        auto market = TestMarket{*app};
-        auto source = root.create("source", minBalance1);
-        auto destination = root.create("destination", minBalance1);
-        source.changeTrust(idr, 20);
-        destination.changeTrust(usd, 20);
-        gateway.pay(source, idr, 10);
-        for_versions_from(12, *app, [&] {
-            auto btc = makeAsset(getAccount("missing"), "BTC");
-            REQUIRE_THROWS_AS(source.pathPaymentStrictSend(
-                                  destination, idr, 10, usd, 10, {btc}, &btc),
-                              ex_PATH_PAYMENT_STRICT_SEND_NO_ISSUER);
-            // clang-format off
-            market.requireBalances(
-                {{source, {{xlm, minBalance1 - 2 * txfee}, {idr, 10}, {usd, 0}}},
-                 {destination, {{xlm, minBalance1 - txfee}, {idr, 0}, {usd, 0}}}});
-            // clang-format on
-        });
-    }
+        SECTION("send issuer missing")
```

Comment:
Same thing is the other tests: it would be better to have a test that actually checks for success. I'm going to stop repeating this comment, but it applies in every case.

Reply:
I modified an existing successful path payment test to also test the no issuer case

Reply:
I like what you did here.